src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -450,6 +450,18 @@ export class DynamicFacet extends Component {
     return allDynamicFacets.indexOf(this) + 1;
   }
 
+  public isCurrentlyDisplayed() {
```

Comment:
👍 , this makes the facet `autoLayoutAdjustable` inside `FacetColumnAutoLayoutAdjustment`.

In short: when all components in the facet column implements this interface, and all components return "isCurrentlyDisplayed()=> false", we change the facet column layout to hide itself automatically so that the rest of the page like the result list takes all the available space in the page.

Would be better though to add `implements IAutoLayoutAdjustableInsideFacetColumn ` on the class definition.

Are the "responsive" options also implemented properly ? 
When you resize to mobile mode, does it tuck itself away properly in a dropdown menu ?